### PR TITLE
W7 - TungNguyen - UI Test

### PR DESCRIPTION
**Pull request by at-tungnguyen2**
#### Related issues (Optional)
Task #100

#### Description (Required)
- [x]  Create UI test

### CODE REVIEW

File: app/src/androidTestAt-tungnguyenDebug/java/vn/asiantech/internship/tungnguyen/LoginUiTest.java [hunk 1]
```diff
@@ -0,0 +1,201 @@
+package vn.asiantech.internship.tungnguyen;
```
Comment (at-dev-support-01) [inline app/src/androidTestAt-tungnguyenDebug/java/vn/asiantech/internship/tungnguyen/LoginUiTest.java]: WARNING: [TooManyStaticImports] Too many static imports may lead to messy code
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#TooManyStaticImports

**Comment by at-tungnguyen2** [comment]
@at-trungnguyen  Please check for me, thank you!

File: app/src/androidTestAt-tungnguyenDebug/java/vn/asiantech/internship/tungnguyen/LoginUiTest.java (hunk 1 above)
Comment (at-dev-support-01) [inline app/src/androidTestAt-tungnguyenDebug/java/vn/asiantech/internship/tungnguyen/LoginUiTest.java]: WARNING: [TooManyStaticImports] Too many static imports may lead to messy code
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#TooManyStaticImports

File: app/build.gradle [hunk 2]
```diff
@@ -100,7 +100,12 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     implementation 'com.android.support:support-v4:26.1.0'
     implementation 'com.romandanylyk:pageindicatorview:0.2.0'
+    implementation 'junit:junit:4.12'
```
Comment (at-trungnguyen) [inline app/build.gradle]: duplicate, remove this

File: app/build.gradle [hunk 3]
```diff
@@ -100,7 +100,12 @@ dependencies {
...
     testImplementation 'org.mockito:mockito-core:2.13.0'
+    androidTestImplementation 'com.android.support.test:runner:1.0.1'
+    androidTestImplementation 'com.android.support.test:rules:1.0.1'
+    androidTestImplementation'com.android.support.test.espresso:espresso-core:3.0.1'
+    androidTestCompile 'com.android.support.test.espresso:espresso-core:3.0.1'
```
Comment (at-trungnguyen) [inline app/build.gradle]: duplicate, remove it

File: app/build.gradle [hunk 4]
```diff
@@ -100,7 +100,12 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     implementation 'com.android.support:support-v4:26.1.0'
     implementation 'com.romandanylyk:pageindicatorview:0.2.0'
+    implementation 'junit:junit:4.12'
     testImplementation 'org.mockito:mockito-core:2.13.0'
+    androidTestImplementation 'com.android.support.test:runner:1.0.1'
```
Comment (at-trungnguyen) [inline app/build.gradle]: duplicate

File: app/build.gradle [hunk 5]
```diff
@@ -100,7 +100,12 @@ dependencies {
...
+    androidTestImplementation 'com.android.support.test:rules:1.0.1'
```
Comment (at-trungnguyen) [inline app/build.gradle]: duplicate

File: app/src/at-tungnguyen/res/values/strings.xml [hunk 6]
```diff
@@ -111,4 +111,5 @@
     <string name="textview_kb">KB/</string>
     <string name="bracket_first">(</string>
     <string name="login_successfully">Login successfully</string>
+
```
Comment (at-trungnguyen) [inline app/src/at-tungnguyen/res/values/strings.xml]: remove line

File: app/src/androidTestAt-tungnguyenDebug/java/vn/asiantech/internship/tungnguyen/LoginUiTest.java (hunk 1 above)
Comment (at-dev-support-01) [inline app/src/androidTestAt-tungnguyenDebug/java/vn/asiantech/internship/tungnguyen/LoginUiTest.java]: WARNING: [TooManyStaticImports] Too many static imports may lead to messy code
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#TooManyStaticImports

**Comment by at-tungnguyen2** [comment]
@at-trungnguyen  Please check for me, thank you

**Pull request merged**